python/taichi/lang/ast/__init__.py

```diff
@@ -1 +1,13 @@
+from taichi.lang.ast.ast_transformer import ASTTransformer
+from taichi.lang.ast.ast_transformer_check import ASTTransformerChecks
+from taichi.lang.ast.ast_transformer_utils import (ASTTransformerContext,
+                                                   print_ast)
 
+
+def transform_tree(tree, ctx: ASTTransformerContext):
```

Comment:
Can we move this to another file? ;) Ideally __init__ files should only contain imports from other files :P